.github/workflows/test-desktop.yml

```diff
@@ -256,3 +256,58 @@ jobs:
                 body,
               })
             }
+
+  allure-report:
+    name: "Allure Reports Export on Server"
+    needs: [test-desktop-app, test-desktop-app-mac-internal, report]
+    runs-on: [ledger-live, linux]
+    if: always()
+    steps:
+      - name: Download Allure Report - Windows
+        uses: actions/download-artifact@v2
+        with:
+          name: allure-results-windows
+          path: allure-results-windows
+      - name: Download Allure Report - MacOS
+        uses: actions/download-artifact@v2
+        with:
+          name: allure-results-macos
+          path: allure-results-macos
+      - name: Download Allure Report - Linux
+        uses: actions/download-artifact@v2
+        with:
+          name: allure-results-linux
+          path: allure-results-linux
+      - name: Send Results and Generate Allure Report - Windows
```

Comment:
You can use this composite [action](https://github.com/LedgerHQ/ledger-live/blob/develop/tools/actions/composites/upload-allure-report/action.yml) (to improve) for sending action to factorise a 'little' bit.

Reply:
Cool will take a look